### PR TITLE
New Custom Touch Controls: Red Dead Redemption 2

### PR DESCRIPTION
**Pull request by evanpaul14**
Screenshot 2024-03-12 at 9 49 05 PM

### CODE REVIEW

**Comment by redphx** [comment]
Thanks for contributing!  

Some feedbacks:
- Please use the full name `red-dead-redemption-2` for file name (and in the JSON file)
- Beautify JSON files

**Comment by evanpaul14** [comment]
It should be good now.

File: touch-layouts/591405932.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "name": "Red Dead Redemption 2",
+  "default_layout": "custom",
+  "layouts": [
+    "red-dead-2"
```
Comment (redphx) [inline touch-layouts/591405932.json]: You forgot to update this

File: touch-layouts/layouts/red-dead-redemption-2.json [hunk 2]
```diff
@@ -0,0 +1,272 @@
+{
```
Comment (redphx) [inline touch-layouts/layouts/red-dead-redemption-2.json:1]: Use this tool to format json (indent = 2): https://duckduckgo.com/?t=ffab&q=format+json&ia=answer

**Comment by redphx** [comment]
Thanks! You can test the new layout now.

**Pull request merged**